app/javascript/components/workflows/index.tsx

```diff
@@ -134,14 +135,16 @@ const Exam: React.FC = () => {
       );
     case 'RESULT':
       return (
-        <ShowExam
-          railsUser={railsUser}
-          railsExam={showRes.response.railsExam}
-          railsCourse={showRes.response.railsCourse}
-          railsRegistration={showRes.response.railsRegistration}
-          final={showRes.response.final}
-          lastSnapshot={showRes.response.lastSnapshot}
-        />
+        <DocumentTitle title={showRes.response.railsExam.name}>
```

Comment:
Does it matter whether `<DocumentTitle>` surrounds its children, or is just the first child of the `<Route>` before any others?

Reply:
Either should work.